ui/components/institutional/note-to-trader/note-to-trader.tsx

```diff
@@ -33,29 +59,31 @@ const NoteToTrader: React.FC<NoteToTraderProps> = ({
           display={Display.Flex}
           justifyContent={JustifyContent.spaceBetween}
         >
-          <Label htmlFor="transaction-note">{labelText}</Label>
+          <Label htmlFor="transaction-note">{t('transactionNote')}</Label>
           <Text className="note-header__counter">
-            {noteText.length}/{maxLength}
+            {noteText.length}/{280}
```

Comment:
I'm not sure what's the intended behaviour with this, but do we want to do `{noteText.length}/280` or `{noteText.length}/{MAX_LENGTH}` with `const MAX_LENGTH = 280`?

Reply:
the 280 is the max length. I'll assign it to a const 👍🏼